modules/button.scss

```
@@ -133,4 +123,48 @@
       }
     }
   }
+
+  @media (min-device-width: 320px) and (max-device-width: 428px) {
+    &--type--serch {
+      width: 158px;
+      height: 58px;
+    }
+
+    &--type--hotels {
+      font-size: 30px;
+      width: 100%;
+    }
+
+    &--type--icon {
+      width: 72px;
+      height: 72px;
```

Можно все атйпы сделать еще круче 
```suggestion
    &--type {
    
      &--serch {}
      
      &--hotels {
      font-size: 30px;
      width: 100%;
    }

    &--icon {
      width: 72px;
      height: 72px;
    
    }
  }
 
```

---

Честно, я не понял изменения.
Ну кроме того, что название класса сократилось, но оно у меня и так сокращенное (хз, почему тут он еще добавляет &--type, у меня в визуалке не добавлено).